src/backend/src/modules/auth/auth.controller.ts

```diff
@@ -45,7 +45,7 @@ export class AuthController {
 
   @Get('user')
   async getUserInfo(@Req() req) {
-    if (req.headers.authorization === 'Bearer') return;
+    if (!req.headers.authorization.startsWith('Bearer')) return;
```

Comment:
i just realize this bug when create test, here should check the authorization start with Bearer instead ===, please le t me know if you have any concerns

Reply:
Out apis are protected by auth guards. User can never reach this statement without a Bearer token.

Reply:
but sure, we can keep it. instead can you please change to `Bearer ` 
Bearer with a space?